conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
Of course you're right that this stuff needs to be documented. I just prefer to first get the code into a mergable state and then document the changes, because until the code is good enough to merge I don't know how much of it will change again and then I'll always need to keep the documentation up2date again.